### PR TITLE
Issue #SB-18262: Adding config for ETB dialcode reports

### PR DESCRIPTION
**Pull request by ishawakankar**

### CODE REVIEW

File: ansible/roles/data-products-deploy/templates/model-config.j2 [hunk 1]
```diff
@@ -96,7 +96,7 @@ config() {
...
..."Level 4 Name","l5Name": "Level 5 Name","dialcode": "QR Code","sum(scans)": "Total Scans","noOfContent": "Number of contents","nodeType": "Type of Node","term": "Term"},"output": [{"type": "csv","dims": ["identifier", "channel", "name"],"fileParameters": ["id", "dims"]}]},"mergeConfig": {"frequency": "WEEK","basePath": "'$baseScriptPath'","rollup": 0,"reportPath": "dialcode_counts.csv"},"etbFileConfig": {"bucket": "{{ bucket }}","file": "druid-reports/etb_metrics/dialcode_counts.csv","filePath": "src/test/resources/reports/dialcode_counts.csv"},"etbFileConfig": {"bucket": "'$bucket'","file": "druid-reports/etb_metrics/dialcode_counts.csv","filePath": "src/test/resources/reports/dialcode_counts.csv"},"esConfig": {"request": {"filters": {"contentType": ["Textbook"],"status": ["Live", "Review", "Draft"]},"sort_by": {"createdOn": "desc"},"limit": 10000}},"tenantConfig": {"tenantId": "","slugName": ""},"store": "azure","format": "csv","key": "druid-reports/","filePath": "druid-reports/","container": "'$bucket'","folderPrefix": ["slug", "reportName"]},"output": [{"to": "console","params": {"printEvent": false}}],"parallelization": 8,"appName": "ETB Metrics Model","deviceMapping": false}'
```
Comment (sowmya-dixit) [inline ansible/roles/data-products-deploy/templates/model-config.j2:99]: "{{ bucket }}" is incorrect use "'$bucket'"

Comment (ishawakankar) [reply ansible/roles/data-products-deploy/templates/model-config.j2]: Done

File: ansible/roles/data-products-deploy/templates/model-config.json.j2 [hunk 2]
```diff
@@ -3207,74 +3207,115 @@
...
+					"reportPath": "dialcode_counts.csv",
+					"rollupAge": "ACADEMIC_YEAR",
+					"rollupCol": "Date",
+					"rollupRange": 1
+				}
+		},
+		"etbFileConfig": {
+			"bucket": "reports",
+			"file": "dialcode_scans/dialcode_counts.csv",
+			"filePath": "src/test/resources/reports/dialcode_counts.csv"
```
Comment (sowmya-dixit) [inline ansible/roles/data-products-deploy/templates/model-config.json.j2:3283]: filePath cannot have "src/test/resources/" - Change it

Comment (ishawakankar) [reply ansible/roles/data-products-deploy/templates/model-config.json.j2]: Done.

**Pull request merged**